packages/PowerGrid/src/tests/webgpu-tests.ts

```diff
@@ -0,0 +1,53 @@
+import * as grok from 'datagrok-api/grok';
+import * as ui from 'datagrok-api/ui';
+import * as DG from 'datagrok-api/dg';
+
+import $ from 'cash-dom';
+
+import {awaitCheck, category, expect, expectArray, test, testEvent} from '@datagrok-libraries/utils/src/test';
+import { _scWebGPURender, _scWebGPUPointHitTest } from '../package';
```

Comment:
generally, this is a recipe for cyclic dependency errors :D 

instead of these, use the native grok func calls and remove imports from package:

const renderFunc = DG.Func.Find({tags: ['webGPURenderTag']})[0]
const hitTestFunc = DG.Func.Find({tags: ['webGPUHitTestTag']})[0]

and then use the arguments in structured way (yes the typing is lost, but build will be smaller and you will avoid any cyclic dependencies)
await renderFunc.apply({sc: sc, show: true})